docs/reference/ml/anomaly-detection/apis/preview-datafeed.asciidoc

```diff
@@ -43,12 +50,32 @@ supply the credentials.
 == {api-path-parms-title}
 
 `<datafeed_id>`::
-(Required, string)
+(Optional, string)
 include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=datafeed-id]
++
+NOTE: If you provide the `<datafeed_id>` as a path parameter, you cannot
+provide {dfeed} or {anomaly-job} configuration details in the request body.
+
+[[ml-preview-datafeed-request-body]]
+== {api-request-body-title}
+
+`datafeed_config`::
+(Optional, object) The {dfeed} definition to preview. For valid definitions, see
+the <<ml-put-datafeed-request-body,create {dfeeds} API>>.
+
+`job_config`::
+(Optional, object) The configuration details for the {anomaly-job} that is
+associated with the {dfeed}. If the `datafeed_config` object does not include a
+`job_id` that references an existing {anomaly-job}, you must supply this
+`job_config` object. If you include both a `job_id` and a `job_config`, the
+latter information is used. For valid definitions, see the
```

Comment:
It seems to be implied that you can't supply *only* a job_config. If that's true, I think we should say so explicitly so there's no doubt:
```suggestion
latter information is used. You cannot specify a `job_config` object unless you also supply a `datafeed_config` object. For valid definitions, see the
```